### DISCUSSION TITLE
Tabbed interface

### DISCUSSION THREAD

**Discussion by Cyberax** [elfmz/far2l#2427] [General]
Есть ли какая-нибудь возможность использовать интерфейс с несколькими табами в FARе?

**Comment by akruphi** [comment]
> Есть ли какая-нибудь возможность использовать интерфейс с несколькими табами в FARе?

В текущий момент нет. А что не хватает в возможности открывать несколько файлов на просмотр/редактирование и переходить между ними и панелью по **Ctrl**+[**Shift**]+**Tab** или по **F12**?

**Reply by Cyberax** [reply]
Мне, в основном, хочется запоминать положение обеих панелей. А в списке по F12 можно ли создать ещё одну строку с панелью?

**Reply by akruphi** [reply]
> А в списке по F12 можно ли создать ещё одну строку с панелью?

При текущем функционале нет - панели только единственные.

**Comment by faerot** [comment]
Я решал єто и в виндовом и в линуксовом фарах запуском нескольких инстансов фара в одном окне терминала (ConEmu / Kitty). Хотекеи на запуски и переключения прописьіваються в конфиге терминалов.
